plinth/modules/matrixsynapse/__init__.py

```diff
@@ -149,3 +151,19 @@ def get_configured_domain_name():
         config, _, _ = load_yaml_guess_indent(config_file)
 
     return config['server_name']
+
+
+def is_user_registrations_enabled():
+    """Return whether user registrations are enabled"""
+    with YAMLFile('/etc/matrix-synapse/homeserver.yaml') as conf:
+        return conf["enable_registration"]
+
+
+def enable_user_registrations():
```

Comment:
It seems a bit unnecessary to me to wrap the single line into another function here.